### PR TITLE
Update CHANGELOG for AAD features

### PR DESCRIPTION
**Pull request by ZhuXiaoBing-cn**
Update CHANGELOG.md in  the azure-spring-boot

### CODE REVIEW

File: sdk/spring/azure-spring-boot/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
 # Release History
 
 ## 3.0.0-beta.2 (Unreleased)
-- Deprecated `AADAppRoleStatelessAuthenticationFilter` and `AADAuthenticationFilter`
+### Breaking Changes
+- Deprecated `AADAppRoleStatelessAuthenticationFilter` and `AADAuthenticationFilter`.
```
[the remark this answers is not in the document]
Comment (ZhuXiaoBing-cn) [reply sdk/spring/azure-spring-boot/CHANGELOG.md:5]: okay

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by saragluna** [comment]
Should we update the changelog of aad starter too?

**Comment by ZhuXiaoBing-cn** [comment]
@saragluna Okay.

**Review by saragluna — CHANGES_REQUESTED** [review]
LGTM.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**